doc/modules/reference.rst

```diff
@@ -277,3 +277,31 @@ uses.
    clean
 
 
+.. _connectivity_ref:
+
+:mod:`nilearn.connectivity`: Functional Connectivity
+====================================================
+
+.. automodule:: nilearn.connectivity
+   :no-members:
+   :no-inherited-members:
+
+**Classes**:
+
+.. currentmodule:: nilearn.connectivity.connectivity_matrices
```

Comment:
when I write only .. currentmodule:: nilearn.connectome in the file reference.rst, my doc is not generated